### PR TITLE
SCC-352 Replace Heroku with Render.com

### PR DESCRIPTION
**Pull request by matsinet**

### CODE REVIEW

File: Section08/8.2-HTTP-Server-using-Express.md [hunk 1]
```diff
@@ -307,18 +316,27 @@ If we want our request bodies to be properly parsed automatically, we need to te
...
-   app
-     .route("/status")
-     .get((request, response) => {
-        response.status(200).json({message: "Service healthy"})
-     })
-     .post((request, response) => {
-       response.json({requestBody: request.body});
-     });
-   ```
+Let's add a couple of number using the add route handler.  This will translate the `numberOne` and `numberTwo` object attributes from the body of the request:
```
Comment (JohanBester) [inline Section08/8.2-HTTP-Server-using-Express.md:319]: The word numbers should probably be plural
like this ...
"Let's add a couple of numbers using the add route handler."

**Review by JohanBester — COMMENTED** [review]
I made a comment on the code in line 319

File: Section09/9-Homework.md [hunk 2]
```diff
@@ -12,6 +12,7 @@
 ## Goal: _Capstone work_
 
 - Setup a MongoDB Atlas Database (online) for your capstone
-- Host the backend on Heroku and connect MongoDB Atlas
-- Complete your backend and link it to your frontend
+- Create at least one API model and route that pertains to your capstone
+- Host the backend on Render.com and connect MongoDB Atlas using a environment variable
+- Connect you hosted API to your capstone SPA
```
Comment (drod1107) [inline Section09/9-Homework.md:17]: Line 17 typo - "Connect you" should be "connect your"

**Review by drod1107 — DISMISSED** [review]

**Review by drod1107 — APPROVED** [review]
Looks great! Thanks for banging this out!

**Pull request merged**